src/we_rsa.c

```diff
@@ -867,6 +1080,81 @@ static int we_rsa_priv_enc_int(size_t fromLen, const unsigned char *from,
                     ret = -1;
                 }
             }
+            break;
+        case RSA_X931_PADDING:
```

Comment:
Make this able to be compiled out?

Reply:
Do you think it should be on or off by default?

Reply:
I made it on by default.